mvpa2/support/afni/lib_prep_afni_surf.py

```diff
@@ -286,19 +311,75 @@ def yesno2bool(d, k): # dict, key
             c['outvol_ext'] = '+' + c['outvol_view']
             c['outvol_fullext'] = c['outvol_ext'] + '.HEAD'
 
-
     return c
 
+
+
+def _ensure_expvol_is_plump(expvol):
+    '''Helper function that raises an error if the experimental volume
+        is oblique, because such volumes are not supported (yet)'''
+    cmd = '3dinfo "%s"' % expvol
+    info = utils.cmd_capture_output(cmd)
```

Comment:
well... sure we can merge, but it would yet be another "broken window" while awaiting some miracle to happen to fix them all up (e.g. still awaiting for a test for #415) :-/  since function is specific for handling this case, ideally it should have its own (regardless of having or not an overall integration test) unit-test to make testing fast and "encapsulated".